tests/NotifierTest.php

```diff
@@ -127,3 +133,90 @@ public function testEnvironmentIsUnset()
         $this->assertFalse(isset($notice['environment']));
     }
 }
+
+class OnErrorTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        error_reporting(E_ALL | E_STRICT);
+
+        $this->notifier = new NotifierMock([
+            'projectId' => 1,
+            'projectKey' => 'api_key',
+        ]);
+        $handler = new \Airbrake\ErrorHandler($this->notifier);
+        $handler->register();
+
+        echo $undefined_var;
+    }
+
+    public function testPostsError()
+    {
+        $notice = $this->notifier->notice;
+        $error = $notice['errors'][0];
+        $this->assertEquals('Airbrake\Errors\Notice', $error['type']);
+        $this->assertEquals('Undefined variable: undefined_var', $error['message']);
+    }
+
+    public function testPostsBacktrace()
+    {
+        $backtrace = $this->notifier->notice['errors'][0]['backtrace'];
+        $this->assertCount(10, $backtrace);
+
+        $wanted = [[
+            'file' => dirname(dirname(__FILE__)).'/tests/NotifierTest.php',
+            'line' => 150,
```

Comment:
We still have to update lines if we edit tests even if tests are in separate files. It just makes such changes independent.

I don't want to split tests because they test 1 component and it is handy to have them in 1 file.